src/pages/About.tsx

```diff
@@ -1,18 +1,18 @@
-import React from 'react';
+import type { FC } from 'react';
 
-export const About: React.FC = () => {
+export const About: FC = () => {
   return (
     <div className="container mx-auto pt-40 pb-10 px-4">
       <div
         className="p-8 rounded-lg shadow-lg max-w-2xl mx-auto border"
         style={{
```

Comment:
Почему бы не использовать tailwind вместо style?
```suggestion
        className="p-8 rounded-lg shadow-lg max-w-2xl mx-auto border bg-bg-secondary border-border"
```
Должно сработать

Reply:
@belosnezhie здесь тоже самое. выделено не совсем то